juicer/service/tahiti_service.py

```diff
@@ -0,0 +1,32 @@
+# -*- coding: utf-8 -*-
+import json
+import logging
+
+import requests
+
+log = logging.getLogger()
+log.setLevel(logging.DEBUG)
+
+
+def query_tahiti_operations(base_url, item_path, token, item_id):
+    headers = {'X-Auth-Token': token}
+    url = '{}/{}/{}'.format(base_url, item_path, item_id)
+
+    log.debug('Querying Tahiti URL: %s', url)
+
+    r = requests.get(url, headers=headers)
+    if r.status_code == 200:
+        return json.loads(r.text)
+    else:
+        raise RuntimeError(
+            u"Error loading storage id {}: HTTP {} - {}".format(item_id,
+                                                                r.status_code,
+                                                                r.text))
+
+
+def get_storage_info(base_url, token, storage_id):
+    return query_tahiti_operations(base_url, 'storages', token, storage_id)
+
+
+def get_data_source_info(base_url, token, data_source_id):
+    return query_tahiti_operations(base_url, '', token, data_source_id)
```

Comment:
I believe you're using base_url here with suffix "operations". Remove operations from base_url and use it in item_path to keep consistency with function get_storage_info.